Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -595,6 +603,7 @@ extension SectionEditorViewController: SectionEditorNavigationItemControllerDele
             showDestructiveDismissAlert()
         } else {
             closeEditor()
+            logCancelEdit()
```

Comment:
Same note as in `closeEditor` about duplicate event logging. It appears closing an unedited editor view results in several duplicate `logAbort` calls.